scripts/mining/kaggle/execution/plotly_mining/check_lib_use.py

```diff
@@ -0,0 +1,33 @@
+import ast
```

Comment:
Let's not include this file for now. The functionality for analyzing/filtering/selecting notebooks should cover this.